governance.md

```diff
@@ -42,6 +42,11 @@ Contributors work on behalf of and are responsible to the larger Project
 Community and we strive to keep the barrier between Contributors and Users as
 low as possible.
 
+Shapely relies on the [GEOS project](https://libgeos.org) and expresses, in
+Python, much of the GEOS API. The sets of contributors for these projects
+overlap, but they are organized separately and have different forms of
+governance.
```

Comment:
@caspervdw what do you think about this addition?

Reply:
I think the core aim of this project should be expressing the GEOS API in Python. Maybe rephrase it slightly:

```
Shapely relies on the [GEOS project](https://libgeos.org) and aims to express its API
in Python. The sets of contributors for these projects
overlap, but they are organized separately and have different forms of
governance
```

Reply:
I think we mostly agree. The GEOS API has some things that are specific to PostGIS, like SRIDs, which we're not expressing. And the project has some Python geospatial interoperability features that have nothing to do with GEOS (`__geo_interface__` for example). To me. it doesn't feel quite right to say that expressing the API is the core aim of this project.